app/src/main/java/com/sunasterisk/moviedb_51/ui/container/ContainerFragment.kt

```diff
@@ -27,14 +33,66 @@ class ContainerFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         initToolBar()
+        binding.bottomNavigation.setOnNavigationItemSelectedListener(this)
+        startingPosition = 0
+        activity?.replaceFragment(
+            R.id.containerFrameLayout, HomeFragment.newInstance(), true, null, 0
+        )
     }
 
+    override fun onNavigationItemSelected(item: MenuItem) =
+        when (item.itemId) {
+            R.id.home -> {
+                loadFragment(HomeFragment.newInstance(), 1)
+                true
+            }
+            R.id.favorite -> {
+                loadFragment(FavoriteFragment.newInstance(), 2)
+                true
+            }
+            else -> false
+        }
+
     private fun initToolBar() {
         view?.toolbar?.let {
             (activity as? MainActivity)?.run { setSupportActionBar(it) }
         }
     }
 
+    private fun loadFragment(fragment: Fragment, position: Int) {
+        activity?.run {
+            when {
+                startingPosition > position -> {
+                    replaceFragment(
+                        R.id.containerFrameLayout,
+                        fragment,
+                        false,
+                        null, 2
+                    )
```

Comment:
khi hiện đã có giá trị mặc định rồi thì không phải truyền tham số nữa 
```suggestion
                startingPosition > position ->  replaceFragment(
                    R.id.containerFrameLayout, 
                    fragment, 
                    AnimationTypes.LEFT_TO_RIGHT
                )
```